App/screens/ListCredentials.tsx

```diff
@@ -1,25 +1,50 @@
 import type { CredentialRecord } from '@aries-framework/core'
+import type { StackNavigationProp } from '@react-navigation/stack'
 
 import { useCredentials } from '@aries-framework/react-hooks'
+import styled, { css } from '@emotion/native'
+import { useTheme } from '@emotion/react'
+import { useNavigation } from '@react-navigation/core'
 import React from 'react'
 import { useTranslation } from 'react-i18next'
 import { FlatList } from 'react-native'
 
-import { backgroundColor } from '../globalStyles'
+import { CredentialListItem } from 'components'
 
-import { CredentialListItem, Text } from 'components'
+const ItemContainer = styled.TouchableOpacity``
+
+const MessageText = styled.View`
+  text-align: 'center';
+  margin: 100px;
+`
 
 const ListCredentials: React.FC = () => {
   const { credentials } = useCredentials()
   const { t } = useTranslation()
+  const navigation = useNavigation<StackNavigationProp<any, any>>()
```

Comment:
Should be typed? Or is it meant to be generic?

Reply:
It is meant to be a work around to a navigation issue. Without it `navigation.navigate()` coughs up a TS issue 👇🏻 

<img width="943" alt="Screen Shot 2021-11-10 at 3 11 49 PM" src="https://user-images.githubusercontent.com/390891/141208644-894f700f-f333-4310-bf5c-c9d2e730dbd8.png">

Reply:
I've looked into this to see if I can put in proper params like <string, any> but it won't take string. Seems to work with `ParamListBase` but I don't know if that is correct or not.

Reply:
@amanji Digging into this a bit. Looks fixable but getting a dependency cycle that may relate to other nav warnings I see. Will convert to draft and try to fix.

Reply:
Fixed.